packages/ui5-info/test/__snapshots__/ui5-version-info.test.ts.snap

```diff
@@ -3113,67 +3113,67 @@ exports[`getUI5Versions filterOptions: returns support info - includeMaintained
     "version": "1.103.0",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.13",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.12",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.11",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.10",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.9",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.8",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.7",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.6",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.5",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.4",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.3",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.2",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.1",
   },
   {
-    "maintained": false,
+    "maintained": true,
     "version": "1.102.0",
   },
   {
-    "maintained": true,
+    "maintained": false,
     "version": "1.101.1",
   },
   {
-    "maintained": true,
+    "maintained": false,
```

Comment:
All these diffs are a bit alarming. 
But based on the test data https://github.com/SAP/open-ux-tools/blob/d773961f479d876872099aa27b6ee324f7480438/packages/ui5-info/test/testdata/official-response.json 
it seems it is OK.